src/nonstandard.jl

```diff
@@ -56,3 +56,21 @@ constructorof(::Type{<:LinRange}) = linrange_constructor
 ### Expr: args get splatted
 # ::Expr annotation is to make it type-stable on Julia 1.3-
 constructorof(::Type{<:Expr}) = (head, args) -> Expr(head, args...)::Expr
+
+### Cholesky
+setproperties(C::LinearAlgebra.Cholesky, patch::NamedTuple{()}) = C
+function setproperties(C::LinearAlgebra.Cholesky, patch::NamedTuple{(:L,),Tuple{L}}) where {L<:LinearAlgebra.LowerTriangular}
```

Comment:
Just
```suggestion
function setproperties(C::LinearAlgebra.Cholesky, patch::NamedTuple{(:L,),Tuple{<:LinearAlgebra.LowerTriangular}})
```
?

Reply:
Just so you're also aware: this needed another `<:`, i.e. `NamedTuple{(:L,),<:Tuple{<:LinearAlgebra.LowerTriangular}}` :)

Reply:
Interesting, the difference between the two is unclear to me.

Reply:
Oh I see got it.